bundles/org.openhab.binding.wlanthermo/src/main/java/org/openhab/binding/wlanthermo/internal/api/esp32/WlanThermoEsp32CommandHandler.java

```diff
@@ -0,0 +1,298 @@
+        if (channelUID.getId().startsWith("channel")) {
+            int channelId = Integer.parseInt(groupId.substring("channel".length())) - 1;
+            if (channel.size() > 0 && channelId < channel.size()) {
+                switch (channelUID.getIdWithoutGroup()) {
+                    case CHANNEL_NAME:
+                        if (command instanceof StringType) {
+                            channel.get(channelId).setName(command.toFullString());
+                            success = true;
+                        }
+                        break;
+                    case CHANNEL_MIN:
+                        if (command instanceof QuantityType) {
+                            channel.get(channelId).setMin(((QuantityType<?>) command).doubleValue());
+                            success = true;
+                        }
+                        break;
+                    case CHANNEL_MAX:
+                        if (command instanceof QuantityType) {
+                            channel.get(channelId).setMax(((QuantityType<?>) command).doubleValue());
+                            success = true;
+                        }
+                        break;
+                    case CHANNEL_ALARM_DEVICE:
+                        if (command instanceof OnOffType) {
+                            BigInteger value;
+                            if (command == OnOffType.ON) {
+                                value = BigInteger.valueOf(channel.get(channelId).getAlarm()).setBit(1);
+                            } else {
+                                value = BigInteger.valueOf(channel.get(channelId).getAlarm()).clearBit(1);
+                            }
+                            channel.get(channelId).setAlarm(value.intValue());
+                            success = true;
+                        }
+                        break;
+                    case CHANNEL_ALARM_PUSH:
+                        if (command instanceof OnOffType) {
+                            BigInteger value;
+                            if (command == OnOffType.ON) {
+                                value = BigInteger.valueOf(channel.get(channelId).getAlarm()).setBit(0);
+                            } else {
+                                value = BigInteger.valueOf(channel.get(channelId).getAlarm()).clearBit(0);
+                            }
+                            channel.get(channelId).setAlarm(value.intValue());
+                            success = true;
+                        }
+                        break;
+                    case CHANNEL_COLOR_NAME:
+                        if (command instanceof StringType) {
+                            channel.get(channelId).setColor(UtilEsp32.toHex(((StringType) command).toString()));
+                            success = true;
+                        }
+                        break;
+                }
+            }
+        } else if (channelUID.getId().equals("pit1")) {
```

Comment:
These channel ids should be constants specified in your WlanThermoBindingConstants file.